datasets/objectenopenbareruimte/dataset.json

```diff
@@ -605,6 +605,262 @@
           }
         }
       }
-    }
+    },
+    {
+        "id": "Terreindeel",
```

Comment:
De conventie voor ids is om camelCase te gebruiken. Dat betekent dat de eerste letter een kleine letter moet zijn.